csharp/ql/src/experimental/ir/implementation/unaliased_ssa/Instruction.qll

```diff
@@ -1331,7 +1331,7 @@ class CheckedConvertOrThrowInstruction extends UnaryInstruction {
  *
  * If the operand holds a null address, the result is a null address.
  *
- * This instruction is used to represent `dyanmic_cast<void*>` in C++, which returns the pointer to
+ * This instruction is used to represent `dynamic_cast<void*>` in C++, which returns the pointer to
```

Comment:
equivalent to https://github.com/github/codeql/pull/10825/commits/9db0c2ca58da795f25424f55172422d8dc7094c3#diff-546a0e7794007683dcf7eec6abc89108b551fc445c071009b996358069e24ee2R1334